sympy/matrices/tests/test_solvers.py

```diff
@@ -460,7 +460,7 @@ def test_gauss_jordan_solve():
                          [w['tau0']], [w['tau1']]])
     assert params == Matrix([[w['tau0']], [w['tau1']]])
     # watch out for clashing symbols
-    x0, x1, x2, _x0 = symbols('_tau0 _tau1 _tau2 tau1')
+    x0, x1, x2, _x0 = symbols('tau00 tau01 tau02 tau1')
```

Comment:
I wonder if `compare` should not be defined in solvers here:
```
name = _uniquely_named_symbol('tau', aug,
            compare=lambda i: str(i).rstrip('1234567890')).name
```
And perhaps `tau_` should be passed for a nicer looking numbered symbol.

Reply:
`'tau'` will be enough since printers intelligently interprete the number to subscript.

```python
>>> from sympy import Symbol, latex, pretty
>>> latex(Symbol('tau0')) == latex(Symbol('tau_0'))
True
>>> pretty(Symbol('tau0')) == pretty(Symbol('tau_0'))
True
```